DEVELOP.md

```diff
@@ -3,20 +3,23 @@
 If you've made it this far, **thank you**! We appreciate your contribution, and hope that this document helps you along the way. If you have any questions or problems, don't hesitate to [file an issue](https://github.com/primer/css/issues/new).
 
 ## Structure
-Primer CSS is published to [npm] as [@primer/css]. Each of Primer CSS's "modules" lives in a subfolder under `src/` with an `index.scss` in it. Generally speaking, the styles are divided into three primary themes:
+Primer CSS is published to [npm] as [@primer/css]. Each of Primer CSS's components lives in a subfolder under `src/` with an `index.scss` in it. Generally speaking, the styles are divided into three primary themes:
+
+* **Base** (`base/`) includes CSS reset, global styles, and typography
+* **Themes** (`color-modes/`) includes color-mode themes from primer-primitives
+* **Core** (`core/`) index file for all components, along with global CSS and utility imports
+* **Support** (`support/`) SCSS variables and mixins for writing Primer CSS (example: spacer vars, breakpoints, font-size and weight)
+* **Utilities** (`utilities/`) SCSS variables, mixins and utility classes (example: `margin` and `padding`)
 
-* **Core** styles (in `core/`) are common dependencies, which include support variables, native element and typography styles, buttons, navigation, tooltips, etc.
-* **Product** styles (in `product/`) are specific to github.com, and include components such as avatars, labels, markdown styles, popovers, and progress indicators.
-* **Marketing** styles (in `marketing/`) are specific to GitHub marketing efforts, including international and event-focused sites as well as the more design-heavy feature pages on github.com. Marketing styles include new colors and button styles, and extend the core typography and whitespace scales.
 
 ### Paths
 Here's what you need to know about how the files are structured in both git and in the published npm module:
 
 * In git, all of the SCSS source files live in the `src/` directory.
 * When published, all of the files in `src/` are "hoisted" to the package root so that you can import, say, utilities with:
 
-    ```scss
-    @import "@primer/css/utilities/index.scss";
+    ```css
```

Comment:
@jonrohan is this accurate? It doesn't look like we ship scss (I might be wrong) and it also looks like dist doesn't contain subfolders now

Reply:
We do ship the scss, it is used in dotcom, this example pointed to https://unpkg.com/browse/@primer/css@18.2.0/utilities/index.scss 

The css version is built and distributed too, but it's not the same import. https://unpkg.com/browse/@primer/css@18.2.0/dist/utilities.css

Reply:
Ah! Maybe we need an example of both then since the directory structure is different